nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -1286,7 +1286,7 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(
 			parentNode->isBlock=false;
 		}
 
-		if ((isInteractive || role == ROLE_SYSTEM_SEPARATOR || name || description.has_value()) && parentNode->getLength() == 0) {
+		if ((isInteractive || role == ROLE_SYSTEM_SEPARATOR || SysStringLen(name) > 0 || (description.has_value() && !description.value().empty())) && parentNode->getLength() == 0) {

```

Comment:
This is challenging to parse given the line length and complexity

```suggestion
		if (
			(
				isInteractive
				|| role == ROLE_SYSTEM_SEPARATOR
				|| SysStringLen(name) > 0
				|| (
					description.has_value()
					&& !description.value().empty()
				)
			)
			&& parentNode->getLength() == 0
		) {
```